lang/ilias_de.lang

```diff
@@ -4909,6 +4909,7 @@ common#:#search_active#:#Aktive Benutzer
 common#:#search_at_current_position#:#An der aktuellen Position
 common#:#search_engine#:#Lesbare .html-URLs
 common#:#search_for#:#Suchen
+common#:#search_field#:#Sucheingabefeld
```

Comment:
Whenever possible avoid the common module. You could can use the search module here.